docs/developers-docs/development-environment/frontend_environment.md

```diff
@@ -37,6 +37,10 @@ Then clone your forked repository locally
 The `master` branch represent the [Bioportal version](https://bioportal.bioontology.org/), if you want to have the [Agroportal version](https://agroportal.lirmm.fr/).
 You have to merge the `ontoportal-lirmm` branch to your `master`, by doing so `git merge ontoportal-lirmm master`
 
+{: .important }
+For Windows users with Docker Desktop using the WSL 2, it is recommended to clone your repository directly in your home directory of your WSL distribution. Cloning it in any Windows folder that is mounted by default in `/mnt/<drive letter>` would result in very poor performance.
```

Comment:
can you add here, the steps to clone the repository directly in the home directory of the WSL distribution ?

Reply:
Added some details. Please tell me if more is needed.

Reply:
NO, it's perfect, thanks.